R/utils_render_rtf_redo.R

```diff
@@ -240,6 +240,134 @@ rtf_table <- function(rows) {
   rtf_raw(paste(unlist(rows), collapse = ""))
 }
 
+# This is the assumed width of the area within page
+# margins in a default Word document
+standard_width_twips <- 9468L
```

Comment:
It'd be great to have an `options()` option to set this value, e.g. for changing the default page size for a whole document.

Reply:
Now added as `gt.rtf_page_width` (equal to `9468L`).